test-cases/longevity/longevity-200GB-48h-network-monkey.yaml

```diff
@@ -13,7 +13,12 @@ instance_type_db: 'i3.2xlarge'
 
 nemesis_class_name: 'NetworkMonkey'
 nemesis_interval: 10
+
 extra_network_interface: true
+# when using multiple network interfaces in the same subnet, we run into the the possibility
+# that the interface we tried to contact does not respond, but the other one does.
+# To prevent that, we use the public IP address to contact the nodes instead.
+ip_ssh_connections: 'public'
```

Comment:
even if on the same subnet, in this case, will Scylla be configured to listen on both ports?

Reply:
I think that the instance will have only one public IP.

Reply:
I think in extra interface case we need to make scylla to bind to that particular interface, no others, instead of using public ip.

Reply:
I don't think the issue is even related to scylla.
I think the issue is that when we're trying to connect the instances via SSH and you have two interfaces on the same network, it's causing issues.